### PR TITLE
cohort 4: Add guide for demos

### PR DESCRIPTION
**Pull request by maxwofford**

### CODE REVIEW

**Comment by maxwofford** [comment]
@jonleung Could I get feedback on anything this might be missing?

**Comment by maxwofford** [comment]
This addresses #223

File: cohort_4/playbook/activities.md [hunk 1]
```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
...
+  order)
+- Each team should start off with something they failed at
+  - Make sure to give context by explaining the basic overview of the app and
+    why this is a failure
+- After presenting something they failed at, they can show off what they made
+
+### Giving feedback
+
+To keep demos running smoothly, feedback is going to be written on sticky notes
+instead of shouted out after the demo.
```
Comment (zachlatta) [inline cohort_4/playbook/activities.md]: Perhaps add a quick explanation of what feedback constitutes and why we're doing it?

File: cohort_4/playbook/activities.md [hunk 2]
```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
...
+
+- Pass out sticky notes and pencils/pens to the hackers.
+- Have the hackers write the number of the team on the sticky note above their
+  feedback (the first team to present is team 1).
```
Comment (zachlatta) [inline cohort_4/playbook/activities.md]: Perhaps clarify that the team number ascends in the order they demo in?

Ex.

> the first team to present is team #1, the second #2, and so on

File: cohort_4/playbook/activities.md [hunk 3]
```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
 
 TODO
 
+## Demos
+
+### Presenting your demo
```
Comment (zachlatta) [inline cohort_4/playbook/activities.md]: Perhaps there could be a more descriptive title? Remember, this is facilitator facing.

One idea may be to get rid of this heading entirely and just have all this info directly under the `## Demos` heading.

File: cohort_4/playbook/activities.md [hunk 4]
```diff
@@ -58,6 +61,31 @@ first floor with the elevators.
...
+
+Demos are the hackers' chance to show off what they have been putting so much
+time and effort into for the past day.
+
+- Whenever someone wants to present they can volunteer to (no set presentation
+  order)
```
Comment (zachlatta) [inline cohort_4/playbook/activities.md]: This is a pretty passive way to say that the order of demos depends on who volunteers.

Perhaps reword this to be a bit more direct?

**Comment by zachlatta** [comment]
Left a couple comments, just my 2 cents.

## 

Zach Latta
Executive Director of hackEDU https://hackedu.us
http://zachlatta.com
https://github.com/zachlatta
https://twitter.com/zachlatta

**Comment by maxwofford** [comment]
@zachlatta Thanks for the comments! Will address soon :-)

**Comment by maxwofford** [comment]
@zachlatta Comments addressed

/cc @jonleung
